cpp/arcticdb/processing/test/test_clause.cpp

```diff
@@ -18,14 +19,11 @@ void segment_scalar_assert_all_values_equal(const arcticdb::ProcessingSegment& s
     slice_and_key.ensure_segment(empty);
     const arcticdb::SegmentInMemory& segment_memory = slice_and_key.segment(empty);
     segment_memory.init_column_map();
+    auto column_index = segment_memory.column_index(name.value).value();
     size_t row_counter = 0;
     for (auto row : segment_memory) {
```

Comment:
It's kind of hard to know what type row is (I personally despise `auto`), but it looks like it's changed so maybe:
```suggestion
    for (const auto& row : segment_memory) {
```

Reply:
It should be `SegmentInMemoryImpl::Row` which contains a pointer and a number.